### PR TITLE
AbemaTV batch 1/3: [AES] add ECB mode

### PR DESCRIPTION
**Pull request by nao20010128nao**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

This PR adds support for ECB mode in aes.py, which is the simplest block cipher mode of operation.

This is mandatory for downloading from AbemaTV.

### CODE REVIEW

**Review by pukkandan — APPROVED** [review]

File: test/test_aes.py [hunk 1]
```diff
@@ -94,6 +97,19 @@ def test_decrypt_text(self):
         decrypted = (aes_decrypt_text(encrypted, password, 32))
         self.assertEqual(decrypted, self.secret_msg)
 
+    def test_ecb_encrypt(self):
+        data = bytes_to_intlist(self.secret_msg)
+        data += [0x08] * (BLOCK_SIZE_BYTES - len(data) % BLOCK_SIZE_BYTES)
```
Comment (pukkandan) [inline test/test_aes.py:102]: shouldn't this be done by the encryptor?

Comment (nao20010128nao) [reply test/test_aes.py:102]: oh
it should

also this doen't break AbemaTVIE, so I'll remove it

Comment (nao20010128nao) [reply test/test_aes.py:102]: this is because length of `self.secret_msg` isn't multiple of 16, but what should I pad in encryption menthod?

Comment (pukkandan) [reply test/test_aes.py:102]: idk. Is there no padding specs for it? If not, let it be

Comment (nao20010128nao) [reply test/test_aes.py:102]: usually PKCS#7 or `#5`, but nothing except CBC does padding in aes.py

**Pull request merged**